src/main/java/com/celements/rights/access/RightsAccessScriptService.java

```diff
@@ -22,6 +23,9 @@
   @Requirement
   IRightsAccessFacadeRole rightsAccess;
 
+  @Requirement
+  XWikiContext xWikiContext;
```

Comment:
1. XWikiContext isn't a component, so this wouldn't work.
2. xWikiContext is not used.
-> please remove